feature/runtime/testing/loader.go

```diff
@@ -0,0 +1,37 @@
+package testing
+
+import lua "github.com/yuin/gopher-lua"
+
+const (
+	ModuleName = "testing"
+)
+
+type State struct {
```

Comment:
The `testing.State` here is used to collect test passes and failures, to be reported by `runtime.State`.

It needs to be passed to `Loader` below, because the `run` module function interacts with it.